Core/src/org/sleuthkit/autopsy/datamodel/AbstractAbstractFileNode.java

```diff
@@ -182,9 +182,10 @@ private void removeListeners() {
                 Pair<Score, String> scorePropAndDescr = getScorePropertyAndDescription(tags);
                 Score value = scorePropAndDescr.getLeft();
                 String descr = scorePropAndDescr.getRight();
-                CorrelationAttributeInstance attribute = getCorrelationAttributeInstance();
+                List<CorrelationAttributeInstance> listWithJustFileAttr = new ArrayList<>();
```

Comment:
We need to check that CR is enabled, since getCorrelationAttributeInstance() did. This applies below as well.

Reply:
Good catch, checked.